security/security-design/shared-assets/kms-import-keys/README.md

```diff
@@ -0,0 +1,83 @@
+# Importing keys into OCI KMS Vaults
+
+Owner: Leon van Birgelen
+
+Key Management Service is an OCI service that stores and manages keys for secure access to resources.
+
+The Oracle Cloud Infrastructure (OCI) [Key Management Service](https://oracle.com/security/cloud-security/key-management/) (KMS) is a cloud-based service that provides centralized management and control of encryption keys for data stored in OCI.
+
+One of the capabilities of OCI KMS is to import Vault Keys and Key Versions, in case you want to "bring your own key" (BYOK). There is [detailed documentation](https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/KeyManagement/Tasks/importingkeys.htm) available on this process but in this example below you will find a brief guide on how to this as it is a tedious and detailed process.
+
+
+# Importing a RSA 2048 Asynchronous key
+
+This example is for an RSA 2048 Asynchronous Key to be imported in OCI Vault. There are also examples for Synchronous Keys and for importing key versions, see the documentation as mentioned above.
+
+## Prerequisites
+
+- Make sure to have a up-to-date version of OpenSSL installed that supports the RSA_OAEP_AES_SHA256 wrapping mechanism. OCI CloudShell is currently based on Oracle Linux 7, which does not have the minimum required version of OpenSSL installed. If you create an OCI Compute based on Oracle Linux 9, it should work immediately.
```

Comment:
Mention the openssl flag -id-aes256-wrap-pad and how to verify the installed version for it.

Reply:
I added a test command to verify the openssl cypher, but it is apparently not possible to see based on installed/supported ciphers, since this is about if the wrapping with this cipher is allowed.